CONTRIBUTING.md

```diff
@@ -87,20 +123,20 @@ Alternatively, you can do most of the process using the command line:
 
 - fork the repository on the github web interface
 
-- clone your fork locally:  
+- clone your fork locally:
```

Comment:
Please revert the changes from this line onward. The double space at the end of a line is a markdown "feature" (albeit a poorly planned one, I admit) to insert a line break without starting a new paragraph. Keeping commands in their own lines improves readability and facilitates copy-pasting.

Also, it's not good practice to make unrelated changes in the same PR/commit (although I admit I'm often guilty of it!)